test/parallel/test-vm-syntax-error-stderr.js

```diff
@@ -23,13 +23,8 @@ p.stderr.on('data', function(data) {
 });
 
 process.on('exit', function() {
-  // chakra engine don't point line in script file
-  if (common.isChakraEngine) {
-    assert(/^SyntaxError: Expected ';'/.test(output));
-  } else {
-    assert(/BEGIN CERT/.test(output));
-    assert(/^\s+\^/m.test(output));
-    assert(/Invalid left-hand side expression in prefix operation/
-      .test(output));
-  }
+  assert(/BEGIN CERT/.test(output));
+  assert(/^\s+\^/m.test(output));
+  assert(/Error:/
```

Comment:
actually, just checked upstream and your change diverges the upstream test case. Can you wrap this error under `common.engineSpecificMessage()`?

---
In reply to: [118543511](https://github.com/nodejs/node-chakracore/pull/263#discussion_r118543511) [](ancestors = 118543511)